test/e2e/framework/timeouts.go

```diff
@@ -0,0 +1,132 @@
+	// PodList is how long to wait for the pod to be listable.
+	PodList time.Duration
+
+	// PodStart is how long to wait for the pod to be started.
+	PodStart time.Duration
+
+	// PodStartShort is same as `PodStart` to wait for the pod to be started, but shorter.
+	// Use it case by case when we are sure pod start will not be delayed.
+	// minutes by slow docker pulls or something else.
+	PodStartShort time.Duration
+
+	// PodDelete is how long to wait for a pod to be deleted.
+	PodDelete time.Duration
+
+	// PodGet is how long to wait for a pod to be got.
+	PodGet time.Duration
+
+	// PodEvent is how much we wait for a pod event to occur.
+	PodEvent time.Duration
+
+	// ServiceStart is how long to wait for a service endpoint to be resolvable.
+	ServiceStart time.Duration
+
+	// Poll is how often to Poll pods, nodes and claims.
+	Poll time.Duration
+
+	// PollShort is the short timeout value in polling.
+	PollShort time.Duration
+
+	// ServiceAccountProvision is how long to wait for a service account to be provisioned.
+	// service accounts are provisioned after namespace creation
+	// a service account is required to support pod creation in a namespace as part of admission control
+	ServiceAccountProvision time.Duration
+
+	// SingleCall is how long to try single API calls (like 'get' or 'list'). Used to prevent
+	// transient failures from failing tests.
+	SingleCall time.Duration
+
+	// NodeReadyInitial is how long nodes have to be "ready" when a test begins. They should already
+	// be "ready" before the test starts, so this is small.
+	NodeReadyInitial time.Duration
+
+	// PodReadyBefore is how long pods have to be "ready" when a test begins.
+	PodReadyBefore time.Duration
+
+	// ClaimProvisionShort is same as `ClaimProvision` to wait for claim to be dynamically provisioned, but shorter.
+	// Use it case by case when we are sure this timeout is enough.
+	ClaimProvisionShort time.Duration
+
+	// ClaimProvision is how long claims have to become dynamically provisioned.
+	ClaimProvision time.Duration
+
+	// RestartNodeReadyAgain is how long a node is allowed to become "Ready" after it is restarted before
+	// the test is considered failed.
+	RestartNodeReadyAgain time.Duration
+
+	// RestartPodReadyAgain is how long a pod is allowed to become "running" and "ready" after a node
+	// restart before test is considered failed.
+	RestartPodReadyAgain time.Duration
+
+	// SnapshotCreate is how long for snapshot to create snapshotContent.
+	SnapshotCreate time.Duration
+
+	// SnapshotDelete is how long for snapshot to delete snapshotContent.
+	SnapshotDelete time.Duration
+
+	// pv.go
+
+	// PVBindingTimeout is how long PVs have to become bound.
+	PVBinding time.Duration
+
+	// ClaimBindingTimeout is how long claims have to become bound.
+	ClaimBinding time.Duration
+
+	// PVReclaimingTimeout is how long PVs have to beome reclaimed.
+	PVReclaiming time.Duration
+
+	// PVDeletingTimeout is how long PVs have to become deleted.
+	PVDeleting time.Duration
```

Comment:
These either need a better name or be merged with some timeouts above (ClaimBinding -> ClaimProvisionShort?, PVDeleting -> rename to PVDelete,...)

Reply:
These are copies of the original values [here](https://github.com/kubernetes/kubernetes/pull/96042/commits/b37e7b701afa1aeb1477b2b95bafe8fdf24e06d9#diff-f9cf7d59f38dc004bb86a00b169c989bf93d49636bdf254bf38909a3d37816e3L41-L52), but I'm OK with changing the names.

Reply:
Let's create the new variables with proper names as Jan suggested. The old ones will eventually be deprecated.